django_socio_grpc/apps.py

```diff
@@ -10,5 +13,6 @@ class DjangoSocioGrpcConfig(AppConfig):
     verbose_name = "Django Socio gRPC"
 
     def ready(self):
-        # Import signals module to connect the signals
-        pass
+        signals.grpc_action_started.connect(reset_queries)
+        signals.grpc_action_started.connect(close_old_connections)
+        signals.grpc_action_finished.connect(close_old_connections)
```

Comment:
add a comment to explain why it's better to close them both at start and finished signals as it can be not obvious for people that don't know how close_old_connections it's working